src/scanner.cc

```diff
@@ -155,13 +156,7 @@ struct Scanner {
         }
         return false;
       } else {
-        if (lexer->lookahead == '{') {
-          advance(lexer);
-          if (lexer->lookahead == '-') {
-            advance(lexer);
-            next_token_is_comment = iswspace(lexer->lookahead);
-          }
-        }
+        next_token_is_comment = advance_matching_and_whitespace(lexer, "{-");
```

Comment:
In your last PR, @rewinfrey pointed out that we probably don't want the whitespace check here. Maybe the whitespace check should be removed from the helper method?

Reply:
I thought about it, but I can't see any difference in the logic here
![ws](https://user-images.githubusercontent.com/4367963/37387814-40405dc6-275e-11e8-8b7d-2fa371ad60fe.png)